### PR TITLE
Update readme with new badges and banner

### PR DESCRIPTION
**Pull request by infinyte7**
## Purpose / Description
Update Readme

## How Has This Been Tested?
https://github.com/infinyte7/Anki-Android/tree/update-readme

## Checklist
_Please, go through these checks before submitting the PR._

- [ x ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ x ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ x ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ x ] You have commented your code, particularly in hard-to-understand areas
- [ x ] You have performed a self-review of your own code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,44 +13,49 @@
...
+- night mode
+- whiteboard 
+- progress widget
+- detailed statistics
+- syncing with AnkiWeb
+- write answers (optional)
+- text-to-speech integration
+- more than 6000 premade decks
+- spaced repetition (supermemo 2 algorithm)
+- supported flashcard contents: text, images, sounds, LaTeX
```
Comment (mikehardy) [inline README.md:29]:
```suggestion
- supported contents: text, images, sounds, MathJax
```

Just saw that this wrapped for me, this should stop the wrapping, and also emphasizes MathJax (the new new) vs LaTeX (the old and tired) :-)

**Comment by infinyte7** [comment]
I think text to left and logo to right will make it more readable. Updating it.

**Comment by infinyte7** [comment]
Updated.

**Comment by infinyte7** [comment]
https://github.com/infinyte7/Anki-Android/blob/update-readme/README.md

**Comment by mikehardy** [comment]
Hello! with the 2.14 release we started accepting donations with the stated goal of accepting money from users (the users' way of saying "Thanks!") to support maintenance and development of AnkiDroid.

We are looking through PRs merged in releases going back to release 2.10 (where we have good record keeping of what's in releases) to let our contributors know.

If you'd like to submit an expense related to your work here, to share in a portion of our hopefully-sustainable monthly disbursement, please read here and do so in the next couple of days:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process

This is our first time handling donations and attempting to compensate anyone with these processes, so please be flexible and patient with us

Thank you for your contribution to AnkiDroid!

**Pull request merged**